static/app/components/events/interfaces/spans/spanBar.tsx

```diff
@@ -714,6 +720,37 @@ class SpanBar extends React.Component<SpanBarProps, SpanBarState> {
     return errors?.length ? <ErrorBadge /> : null;
   }
 
+  renderEmbeddedTransactionsBadge(
+    transactions: QuickTraceEvent[] | null
+  ): React.ReactNode {
+    const {toggleEmbeddedChildren, organization} = this.props;
+
+    if (!organization.features.includes('unified-span-view')) {
+      return null;
+    }
+
+    if (transactions && transactions.length === 1) {
```

Comment:
This will only support spans with 1 child transaction at the moment right?

Reply:
That's right. I aim to support the case of 2 or more children in a follow up pull request once this prototype validates well.